quiz.rb

```diff
@@ -7,7 +7,11 @@
 # it should return false if age is less than minimum_age
 
 def is_of_age? age, minimum_age
-  # your code goes here
+  if age.to_f >= minimum_age.to_f
```

Comment:
`age.to_f >= minimum_age.to_f` will evaluate to true or false, so you don't need the `if ... else` statement